doc/rst/technotes/reduceIntents.rst

```diff
@@ -72,6 +72,8 @@ For a user-defined reduction, there is a task-private instance
 of the reduction class for each task created for the forall
 or coforall loop. Here is an example of such a class:
 
+.. _readme-reduceIntents-interface:
```

Comment:
I think it would make sense to also create its own section header here, so readers jumping into this point in the document aren't lost about what paragraph to start looking at